src/jqLite.js

```diff
@@ -519,29 +521,34 @@ function jqLiteDocumentLoaded(action, win) {
   }
 }
 
+function jqLiteReady(fn) {
+  var fired = false;
+
+  function trigger() {
+    if (fired) return;
+    fired = true;
```

Comment:
Doubt it.  I'd imagine it would be considered a browser bug if any browsers behaved like that, otherwise you also couldn't add a handler for `load` in `DOMContentReady` which is almost certainly expected to work.

There's also precedent since this is how jQuery does it, so it's pretty battle tested.